From: Masahiro Yamada <yamada.masahiro@socionext.com>
Date: Fri, 17 May 2019 16:54:27 +0900
Subject: s390: drop meaningless 'targets' from tools Makefile
Git-repo: git://git.kernel.org/pub/scm/linux/kernel/git/kvms390/vfio-ccw#vfio-ccw
Git-commit: 10077c9f2dae1afabab2808a0326ecf3e8e5a82c
Patch-mainline: Queued in subsystem maintainer repository
References: s390 kmp build fix

'targets' should be specified to include .*.cmd files to evaluate
if_changed or friends.

Here, facility-defs.h and dis-defs.h are generated by filechk.

Because filechk does not generate .*.cmd file, the 'targets' addition
is meaningless. The filechk correctly updates the target when its
content is changed.

Signed-off-by: Masahiro Yamada <yamada.masahiro@socionext.com>
Signed-off-by: Heiko Carstens <heiko.carstens@de.ibm.com>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 arch/s390/tools/Makefile |    1 -
 1 file changed, 1 deletion(-)

--- a/arch/s390/tools/Makefile
+++ b/arch/s390/tools/Makefile
@@ -6,7 +6,6 @@
 kapi := arch/$(ARCH)/include/generated/asm
 kapi-hdrs-y := $(kapi)/facility-defs.h $(kapi)/dis-defs.h
 
-targets += $(addprefix ../../../,$(kapi-hdrs-y))
 PHONY += kapi
 
 kapi:	$(kapi-hdrs-y)
